tests/utils.go

```diff
@@ -243,6 +243,9 @@ type ObjectEventWatcher struct {
 	dontFailOnMissingEvent bool
 }
 
+const PromptExpression = "(\\$ |\\# )"
```

Comment:
btw you can use const block if you want, but its just a nit

Reply:
Another nit: this should probably be a raw string (`` `(\$ |\# )` ``)

Reply:
How would it work? The ( ) and | are part of the regex.

Reply:
We want the regex string to be `(\$ | \#)`, since both `$` and `#` have to be escaped for the regexp engine to see them as characters.
However, interpreted strings (strings in double-quotes) interpret `\` as an escape character. Therefore, to have a `\` in the string, we need to escape it... Hence the horrible `\\`.
Raw strings (strings in backticks), were created to solve that kind of problem. In raw strings, every character (`(`, `\`, or anything else) is interpreted as such, what you see is what you get.
The downside of raw strings of course is that we can't escape anything inside them, by definition. Things like `\n` would result in those 2 characters being in the string, not a line feed. But in this case we're good.

Reply:
Thanks for the great explanation! Done (and pushed:))